app/mailers/rent_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class RentMailer < ApplicationMailer
+  def rent_email
+    @rent = params[:rent]
+    mail(
+      to: @rent.user.email,
+      subject: "Tanks for rent #{@rent.user.first_name} #{@rent.user.last_name}"
```

Comment:
"Thanks for your rent #{@rent.user.first_name} #{@rent.user.last_name}!"
There is a typo :P